pom.xml

```diff
@@ -71,6 +71,14 @@
                         </dependency>
                     </dependencies>
                 </plugin>
+				<plugin>
+					<groupId>org.codehaus.mojo</groupId>
+					<artifactId>findbugs-maven-plugin</artifactId>
```

Comment:
this also isn't what you want, try updating the parent pom to the latest version, https://github.com/jenkinsci/plugin-pom/releases/tag/plugin-4.13

see archetypes for an example of a standard setup:
https://github.com/jenkinsci/archetypes/blob/master/empty-plugin/src/main/resources/archetype-resources/pom.xml

Reply:
Thanks, I shall have a play around today